dimod/binary/cybqm/cybqm_template.pxd.pxi

```diff
@@ -72,6 +72,7 @@ cdef extern from "dimod/quadratic_model.h" namespace "dimod" nogil:
         size_type num_interactions(index_type)
         bint remove_interaction(index_type, index_type) except +
         void resize(index_type)
+        void scale(cython.double)
```

Comment:
since the underlying C++ code takes `bias_type`, you should just use `bias_type` here as well. The compiler will figure it out either way but no point making it harder for it :smile:

Reply:
I had a feeling to go that way but wasn't sure if that would cause any issues. If it's not an issue, then it is preferable to use the template type alias.